discover/portal/articles/07-collaborating/01-publishing-blogs.markdown

```diff
@@ -0,0 +1,444 @@
+ 
+![Figure 2: Fullscreen view helps you minimize screen clutter and render HTML changes in real time.](../../images/blogs-full-screen-editor.png)
+
+Every 25 seconds, the entry you're editing is automatically saved as a draft; so
+a browser crash or network interruption won't cause you to lose your entry. You
+can exit the fullscreen view, by clicking on *Done* (which saves your content)
+or clicking on *Cancel* to abandon any changes since the last auto-save. From
+the normal sized source view, you can click the `abc` icon to switch back to
+regular view to use the WYSIWYG editor. 
+
+You've learned all the ins and outs of the content editor. Next, you'll learn
+how to specify your blog entry's other characteristics. 
+
+The edit screen's first input field--*Drag \& Drop to Upload*--lets you add a
+cover image (optional) for your entry. This might be an image that represents
+your entry's purpose and attract readers. You can drag and drop onto this field
+any image you like. As an alternative to dragging and dropping an image, you
+click on the *Select File* button to use the image selector to choose an
+existing image in the blog, an image file from Documents and Media, or an image
+outside the portal. You can browse to a file and upload it, or you can specify
+an image's URL for the image selector to take in the image automatically. After
+you set the entry's cover image, you can add a caption to it. And if you want to
+select a different image, you can click the *Change* icon (circling arrows) to
+bring up the image selector again.
+
+Clicking on the *Configuration* icon (gear) at the top of the entry editor
+displays lets you configure the blog entry. It organizes the configuration
+options under several headings. Its Display Date heading reports the date and
+time the entry was first published. The Abstract section lets you choose between
+using 400 character long text-only abstracts or a Custom Abstract. Custom
+Abstract enables you to represent your entry using a thumbnail image and
+description. The Email Notification section let's you enable sending emails on
+entry updates. 
+
+The Categorization heading is next in the entry configuration screen. Its
+options allow you to attribute tags and/or categories to your blog entry. You
+should definitely consider doing this: it improves search results for blog
+entries, and it gives you more navigation options that you can pass on to your
+users. For example, you can add the Tags Navigation portlet to another column on
+your blogs page, allowing users to browse blog entries by tag. 
+
+Below this is the Related Assets heading. If there's some other content in the
+portal that's related to your entry, you can choose it here. For example, you
+might want to write a blog entry talking about a particular discussion that
+happened on the forums. To link those two assets together, select the forum
+thread under Related Assets. 
+
+Blog entries also support trackbacks and pingbacks. Trackbacks are
```

Comment:
Regarding trackbacks, we have disabled those by default in portal.properties (they are really all and most of the new blogs don't support it anymore). Maybe we sholdn't include it here, or at least indicate that is deprecated?
We support pingbacks though, so maybe we can give more relevance to Pingback than trackbacks?